### ISSUE TITLE
database "phylophlan" not found in "phylophlan_databases"

### ISSUE THREAD

**Issue by mcahn** [biobakery/phylophlan#8]
I've installed phylophlan (the version in today's github repo (2020-04-21), and I'm attempting to run Example-02:-Tree-of-life.

I've run phylophlan_get_reference, and created the config file:

`
phylophlan_write_config_file -d a \
    -o 02_tol.cfg \
    --db_aa diamond \
    --map_dna diamond \
    --map_aa diamond \
    --msa mafft \
    --trim trimal \
    --tree1 iqtree \
    --verbose 2>&1 | tee phylophlan_write_config_file.log
`

When I run phylophlan:

`
phylophlan -i input_genomes \
    -d phylophlan \
    -f 02_tol.cfg \
    --diversity high \
    --fast \
    -o output_tol \
    --nproc 8 \
    --verbose 2>&1 | tee logs/phylophlan.log
`

I get this error:

`
[e] database "phylophlan" not found in "phylophlan_databases"
Available databases in "phylophlan_databases":
`

Am I supposed to put something in phylophlan_databases before I run this example?  Bear with me, I'm a sysadmin rather than a biologist.

Thanks,
Matthew Cahn

**Comment by fasnicar** [comment]
Dear Matthew,

Thanks for reporting this. Since you specified `-d phylophlan` you should not put anything inside the `phylophlan_databases` folder, as it should be automatically downloaded.
I think I did a fix yesterday that broke this part. It should now be fixed b4ccc17f5f8af864bcacfd72dea72b8cd1c73822. 
Can you update the code and try again?

Many thanks,
Francesco

**Comment by mcahn** [comment]
Hi Francesco,

I think this is fixed.  I did a "git pull", moved the phylophlan_databases directory out of the way, and ran the job again.  This time phylophlan created the directory, but failed to do the download -- almost certainly because the compute nodes on our cluster can't reach out to the Internet.  So in this environment the download would have to be done manually the way I did it before.  Just for the record the error message was:

Downloading "https://www.dropbox.com/s/x7cvma5bjzlllbt/phylophlan_databases.txt?dl=1" to "phylophlan_databases/phylophlan_databases.txt"
[e] unable to download "https://www.dropbox.com/s/x7cvma5bjzlllbt/phylophlan_databases.txt?dl=1"

Thanks very much for your help,
Matthew
